src/main/scala/mesosphere/marathon/api/TaskKiller.scala

```diff
@@ -25,7 +25,7 @@ class TaskKiller @Inject() (
     val config: MarathonConf,
     val authenticator: Authenticator,
     val authorizer: Authorizer,
-    killService: KillService) extends AuthResource with StrictLogging {
+    killService: KillService)(implicit val executionContext: ExecutionContext) extends AuthResource with StrictLogging {
```

Comment:
Where are we using the execution context?

Reply:
AuthResource requires it.